File: CheckMyStar.Angular/src/app/www/20_Models/BackOffice/Folder.model.ts
import { AccommodationModel} from './Accommodation.model';
import { UserModel } from '../Common/User.model';
import { FolderStatusModel } from './FolderStatus.model';
import { QuoteModel} from './Quote.model';
import { InvoiceModel } from './Invoice.model';
import { AppointmentModel } from './Appointment.model';

export interface FolderModel {
  identifier: number;
  accommodation?: AccommodationModel;
  owner?: UserModel;
  inspector?: UserModel;
  folderStatus?: FolderStatusModel;
  quote?: QuoteModel;
  invoice?: InvoiceModel;
  appointment?: AppointmentModel;
  isActive: boolean;
  updatedDate: Date;
}

File: CheckMyStar.Bll.Models/FolderModel.cs
namespace CheckMyStar.Bll.Models
{
    public class FolderModel
    {
        public int Identifier { get; set; }
        public AccommodationModel? Accommodation { get; set; }
        public UserModel? Owner { get; set; }
        public UserModel? Inspector { get; set; }
        public FolderStatusModel? FolderStatus { get; set; }
        public QuoteModel? Quote { get; set; }
        public InvoiceModel? Invoice { get; set; }
        public AppointmentModel? Appointment { get; set; }
        public bool IsActive { get; set; }
        public DateTime? CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
    }
}
